packages/backend/src/plugins/scaffolder.ts

```diff
@@ -68,7 +68,11 @@ export default async function createPlugin({
       ) as RepoVisibilityOptions;
 
       const githubToken = githubConfig.getString('token');
-      const githubClient = new Octokit({ auth: githubToken });
+      const githubBaseUrl = githubConfig.getString('baseUrl');
```

Comment:
Might be better to use `host` here, as we'll eventually merge this in with the `integrations` config, which already switches based on host.

Tbh I also think it makes sense to default this to `github.com`, so `githubConfig.getOptionalString('host') ?? 'github.com'`.

Reply:
I have renamed the `baseUrl` property to `host` and add github.com as default. 

My first tough was that it could be confusing because in the gitlab config section the similar property is `baseUrl`.  But now I think it's fine.

Reply:
I don't know why the e2e test on windows is failing. Do you have an idea?

Reply:
Retriggered - let's see if it works better